apps/aestratum/priv/Payout.aes

```diff
@@ -0,0 +1,11 @@
+contract Payout =
+
+  public function payout(xs : list((address, int))) : int =
```

Comment:
What is impact if list is long?

Reply:
We don't allow it and split payments for large amount of people to batches of 1000.